### PR TITLE
OCPBUGS-30009: Updated the Network Obser. Welcome page note

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.15, 4.16

Issue:
[OCPBUGS-30009](https://issues.redhat.com/browse/OCPBUGS-30009)

Link to docs preview:
[Network Observability](https://72201--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/#observe-a-cluster)

Additional information:

### CODE REVIEW

File: welcome/index.adoc [hunk 1]
```diff
@@ -300,7 +300,7 @@ Use the Cluster Version Operator (CVO) to upgrade your {product-title} cluster.
...
-- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can view dashboards and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
+- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can view dashboards and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting. After you can generate metrics from flow logs, you can use the metrics when setting custom alerts and creating viewing dashbaords; see xref:../network_observability/metrics-alerts-dashboards.html[Using metrics with dashboards and alerts].
```
Comment (skrthomas) [inline welcome/index.adoc:303]:
```suggestion
- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can xref:../network_observability/metrics-alerts-dashboards.html[view dashboards, customize alerts], and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting. 
```

Comment (skrthomas) [reply welcome/index.adoc:303]: I was thinking more like this. Wdyt?

Comment (dfitzmau) [reply welcome/index.adoc]: HI @skrthomas . Yes, that would work better. Updated.

Comment (skrthomas) [reply welcome/index.adoc]: awesome!! lgtm! Thank you!

**Review by skrthomas — APPROVED** [review]
/lgtm

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/label peer-review-done
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/lgtm

File: welcome/index.adoc [hunk 2]
```diff
@@ -300,7 +300,7 @@ Use the Cluster Version Operator (CVO) to upgrade your {product-title} cluster.
...
-- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can view dashboards and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
+- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can xref:../network_observability/metrics-alerts-dashboards.html[view dashboards, customize alerts], and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
```
Comment (michaelryanpeter) [inline welcome/index.adoc:303]:
```suggestion
- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can xref:../network_observability/metrics-alerts-dashboards.adoc#<anchor_id>_<context>
[view dashboards, customize alerts], and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
```

Comment (michaelryanpeter) [inline welcome/index.adoc:303]: - **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can xref:../network_observability/metrics-alerts-dashboards.adoc#<anchor_id>_<context>
[view dashboards, customize alerts], and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.

**Review by michaelryanpeter — COMMENTED** [review]
You have an incomplete `xref`.

/remove-label merge-review-needed
/remove-label merge-review-in-progress

File: welcome/index.adoc (hunk 2 above)
Comment (michaelryanpeter) [inline welcome/index.adoc:303]:
```suggestion
- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can xref:../network_observability/metrics-alerts-dashboards.adoc#<anchor_id>_<context>
[view dashboards, customize alerts], and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
```

Comment (dfitzmau) [reply welcome/index.adoc:303]: Hi @michaelryanpeter . Updated the link. OK to merge?

**Pull request merged**